src/Template/Bake/Controller/controller.ctp

```diff
@@ -0,0 +1,28 @@
+<%
+use Cake\Utility\Inflector;
+
+$defaultModel = $name;
+%>
+<?php
+namespace <%= $namespace %>\Controller<%= $prefix %>;
+
+use <%= $namespace %>\Controller\AppController;
+
+/**
+ * <%= $name %> Controller
+ *
+ * @property \<%= $namespace %>\Model\Table\<%= $defaultModel %>Table $<%= $defaultModel %>
+<%
+foreach ($components as $component):
+    $classInfo = $this->Bake->classInfo($component, 'Controller/Component', 'Component');
+%>
+ * @property <%= $classInfo['fqn'] %> $<%= $classInfo['name'] %>
+<% endforeach; %>
+ */
+class <%= $name %>Controller extends AppController
+{
+<%
+echo $this->Bake->arrayProperty('helpers', $helpers, ['indent' => false]);
```

Comment:
> Should the templates deviate from the default bake templates?

Maintaining consistency with default bake templates is not necessary. FOC is more opinionated which it was it also has it's own app skeleton.